src/oidc/strategy.js

```diff
@@ -0,0 +1,99 @@
+import { UserManager } from 'oidc-client';
+
+const EVENTS = [
+  'userLoaded',
+  'userUnloaded',
+  'accessTokenExpiring',
+  'accessTokenExpired',
+  'silentRenewError',
+  'userSignedOut'
+];
+
+const ADD_EVENTS = new Map(EVENTS.map(event => ([event, `add${uppercaseFirstLetter(event)}`])));
+const REMOVE_EVENTS = new Map(EVENTS.map(event => ([event, `remove${uppercaseFirstLetter(event)}`])));
```

Comment:
I think we should add prefix to specify those events come from OIDC handling: `openpaas:auth:client:remove${uppercaseFirstLetter(event)}`

Reply:
This mapping is done to not have to call the oidc-client functions explicitely, check the code below